app/tab_deep_dive_countries.py

```diff
@@ -84,8 +84,24 @@ def content_deep_dive_countries(api: PtxboaAPI, res_costs: pd.DataFrame) -> None
         x = None
         st.markdown("TODO: fix surplus countries in data table")
 
-    c1, c2 = st.columns(2, gap="medium")
-    with c2:
+    st.subheader("Full load hours of renewable generation")
+    c1, c2 = st.columns([2, 1], gap="large")
+    with c1:
+        st.markdown("**Map of input data values**")

```

Comment:
Set title to ``Map:``

Reply:
I would use the same titles I suggested here on the input data tab as well

Reply:
I adjusted the titles as suggested and removed the colons as I think a title should go without a colon.